onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_tunable_op.h

```diff
@@ -20,18 +20,19 @@ namespace rocm {
 
 template <typename T>
 struct SkipLayerNormParams : OpParams {
-  SkipLayerNormParams(RocmTuningContext* tuning_ctx, hipStream_t stream, T* output, const T* input,
+  SkipLayerNormParams(RocmTuningContext* tuning_ctx, hipStream_t stream, T* output, T* skip_input_bias_add_output, const T* input,
                       const T* skip, const T* gamma, const T* beta,
                       const T* bias, float epsilon, int ld, int element_count)
-      : OpParams(tuning_ctx, stream), output(output), input(input), skip(skip), gamma(gamma), beta(beta), bias(bias),
-        epsilon(epsilon), ld(ld), element_count(element_count) {}
+      : OpParams(tuning_ctx, stream), output(output), skip_input_bias_add_output(skip_input_bias_add_output), input(input), skip(skip), 
+        gamma(gamma), beta(beta), bias(bias), epsilon(epsilon), ld(ld), element_count(element_count) {}
 
   std::string Signature() const override {
     std::string sig = std::to_string(ld) + "_" + std::to_string(element_count);
     return sig;
   }
 
   T* output;
+  T* skip_input_bias_add_output;
```

Comment:
@PeixuanZuo @ytaous If this will do **inplace update** for this buffer, then you will need to be much more carefully about it. You need to implement the `PreTuning` and `PostTuning` for the TunableOp otherwise the tuning process will garbage fill this buffer.

Reply:
Not familiar with the TunableOp, I'll check with @PeixuanZuo, thx for the review.

Reply:
It's not inplace update there. the formula is `skip_input_bias_add_output = input + skip, output = layernorm(skip_input_bias_add_output + bias)`.

Reply:
It's not inplace update here, the formula is `skip_input_bias_add_output = input + skip + bias, output = layernorm(skip_input_bias_add_output)`.